src/components/MessagesList/MessagesGroup/Message/MessageItem.vue

```diff
@@ -11,41 +11,57 @@
 		:data-next-message-id="nextMessageId"
 		:data-previous-message-id="previousMessageId"
 		class="message"
-		:class="{ 'message--hovered': showMessageButtonsBar }"
+		:class="{
+			'message--hovered': showMessageButtonsBar,
+			'message--sided': isSplitViewEnabled,
+			'message--small-view': (isSmallMobile || isSidebar) && isSplitViewEnabled,
+		}"
 		tabindex="0"
 		@mouseover="handleMouseover"
 		@mouseleave="handleMouseleave">
 		<div
 			:class="{
-				'normal-message-body': !isDeletedMessage,
+				'normal-message-body': !isDeletedMessage && !isSplitViewEnabled,
+				outgoing: actorStore.checkIfSelfIsActor(message) && isSplitViewEnabled,
+				incoming: !actorStore.checkIfSelfIsActor(message) && isSplitViewEnabled,
```

Comment:
These classes are already defined on MessagesGroup, we can use from (with deep styles or non-scoped)